bootstrap/kubeadm/api/v1beta1/kubeadmconfig_webhook.go

```diff
@@ -107,7 +107,7 @@ func (c *KubeadmConfigSpec) validateFiles() field.ErrorList {
 				allErrs = append(
 					allErrs,
 					field.Invalid(
-						field.NewPath("spec", "files").Index(i).Child("contentFrom", "secret", "name"),
+						pathPrefix.Child("files").Index(i).Child("contentFrom", "secret", "name"),
```

Comment:
Not a blocker by any means, but what do you think about moving some of these to field.Required while we're making changes? The error seems more in line with what's being checked here ( and elsewhere for missing values)

Reply:
Sounds good

Reply:
Sounds  good to me - I'll do a bit of a scan and open an issue around it.

Reply:
Perfect, thank you!

Reply:
#6249 covers this I think.

Reply:
Thx!